public/components/trace_analytics/requests/traces_request_handler.ts

```diff
@@ -230,15 +228,35 @@ const hitsToSpanDetailData = async (hits: any, colorMap: any, mode: TraceAnalyti
   };
   if (hits.length === 0) return data;
 
-  const minStartTime = mode === 'jaeger' ? microToMilliSec(hits[hits.length - 1].sort[0]) : nanoToMilliSec(hits[hits.length - 1].sort[0]);
+  const minStartTime =
+    mode === 'jaeger'
```

Comment:
Minor: Do we want to move all these mode checking to the top like: isJaeger = mode === 'jaeger' and replace all mode checkings to isJaeger ? ...

Reply:
good idea but i'll leave refactors out from this bug fix PR, there are a lot more `mode === 'jaeger'` checks